README/README.zh.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://xmcl.app" target="_blank">
-    <img alt="Logo" width="100" src="xmcl-electron-app/icons/dark@256x256.png">
+    <img alt="Logo" width="100" src="https://github.com/BANSAFAn/x-minecraft-launcher/blob/master/README/dark%40256x256.png">
   </a>
```

Comment:
This url should be changed, since you're merging. It should be like `README/dark@256x256.png`?

Reply:
The problem is that when you look at the translated images, the images are not the same, it’s simply not visible, although there is a link there, I honestly can’t figure out how to solve this

Reply:
Google translator translated it wrong.**


Images are not visible in translations  
Only in READMY.md and that's all, the rest are not visible